src/components/tabs/tabview.coffee

```diff
@@ -158,7 +158,7 @@ module.exports = class KDTabView extends KDScrollView
       else if firstPane = @getPaneByIndex 0
         @showPane firstPane
 
-    @emit "PaneRemoved", { pane, handle }
+    @emit "PaneRemoved", { pane, handle, isDetached: shouldDetach }
     return { pane, handle }
 
   removePaneByName:(name)->
```

Comment:
Thanks @gokmen for review and comment. I talked with @sinan and figured out that if pane is detached there won't be `pane.parent` reference and I can use it without touching the framework code. Emitting `isDetached` in `PaneRemoved` event doesn't make sense much. I will let you know if it works.